src/main/webapp/script.js

```diff
@@ -104,20 +104,36 @@ function createParagraphElement(text) {
 // This function returns a message based on sentiment scores
 function getScoreResponse(score) {
   // Messages based on score ranges
-  //TODO: update messages to be something more readable
```

Comment:
Remove this TODO since this is done, same with below

Reply:
Great catch. Just updated it.